Sources/Tools/MaxComponent/plMultistageBehComponent.cpp

```diff
@@ -499,7 +499,7 @@ class MaxStream : public hsStream
         ULONG numRead = 0;
         hsAssert(fLoad, "No Max ILoad!");
         if (fLoad)
-            fLoad->Read(buffer, byteCount, &numRead);
+            fLoad->READ_VOID_BUFFER(buffer, byteCount, &numRead);
```

Comment:
I'm a little uncomfortable with this solution because `READ_VOID_BUFFER` looks like a member function. I think it would be better to redefine `READ_VOID_BUFFER` such that it's more clear that is not a class method such that the usage would be `READ_VOID_BUFFER(fLoad, buffer, byteCount, &numRead);`